{
    "type": "bundle",
    "id": "bundle--0047f190-1e22-47af-a502-9aad1757ee93",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--24b4400b-701c-4ebc-82b3-6f1b3a4963e8",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-03-13T20:10:11.253351Z",
            "modified": "2021-03-13T20:10:11.253351Z",
            "name": "CVE-2012-5642",
            "description": "server/action.py in Fail2ban before 0.8.8 does not properly handle the content of the matches tag, which might allow remote attackers to trigger unsafe behavior in a custom action file via unspecified symbols in this content.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2012-5642"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}